plugin.audio.navahang/main.py

```diff
@@ -43,7 +43,7 @@
 CWD = ADDON.getAddonInfo('path')
 LANGUAGE = ADDON.getLocalizedString
 
-DataLink = 'http://navahang.com/webservice/GetMp3?type=Featured&from=0&to=50'
+DataLink = 'https://navaapi1.b-cdn.net/navaapi2/GetMp3?type=Featured&from=0&to=50'

```

Comment:
This is probably resolved by the load balancer (old url). This might work now but you're likely creating instability in the addon (will break sooner than before).

Reply:
The previous URL is still working, but I changed the URL because it is blocked in Iran.
I will add a server setup option in the new version.